doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -615,6 +615,10 @@ Modem libraries
 
   * Removed references to HERE location services.
 
+* :ref:`lib_at_host` library:
+
+  * Fixed a bug where AT responses would erroneously be logged on the logging UART instead of being written to the chosen ``ncs,at-host-uart`` UART device when the :kconfig:option:`CONFIG_LOG_BACKEND_UART` Kconfig option was set.
```

Comment:
I think this is preferable
```suggestion
  * Fixed a bug where AT responses would erroneously be written to the logging UART instead of being written to the chosen ``ncs,at-host-uart`` UART device when the :kconfig:option:`CONFIG_LOG_BACKEND_UART` Kconfig option was set.
```
@peknis

Reply:
Yes, that sounds better. I was thinking of having logged and logging in the same sentence, but I let it go.

Reply:
Updated, Thanks!